jetty-ee9/jetty-ee9-nested/src/main/java/org/eclipse/jetty/ee9/nested/ContextHandler.java

```diff
@@ -1590,6 +1590,7 @@ public Resource newResource(URI uri) throws IOException
      */
     public Resource newResource(String uriOrPath) throws IOException
     {
+        // TODO WRONG FACTORY
```

Comment:
We still need to figure out what to do here.

Reply:
I think it is actually correct (just badly named) it is a convenience to get a resource from the contexts resource factory, which is not from the contexts resource base!!!!